content-src/components/TopSites/TopSites.scss

```diff
@@ -16,6 +16,23 @@
     @media (min-width: $break-point) {
       display: flex;
     }
+
+    // Only display a placeholder version (ie just outlines/shapes), for use
+    // before sufficient data is available to display.
+    &.placeholder {
+      .site-icon-wrapper {
+        width: 32px;
+        height: 74px;
+      }
+
+      .site-icon-title {
+        border-top: 2px solid $placeholder-grey;
```

Comment:
This line is thicker and lighter than that of the tile. It should probably just be the same:

`1px solid $faintest-black`
https://github.com/mozilla/activity-stream/blob/c9286d83db030fa7a99e8b0c9de6643d34589a52/content-src/main.scss#L45